### DISCUSSION TITLE
Hello. Is everything okay ???

### DISCUSSION THREAD

**Discussion by czarnyafgan** [Try/OpenGothic#197] [General]
Hello guys. First of all - I don't want to sound demanding type. I am a huge fan of OpenGothic project, so much that I come in here every day to check if anything new has appeared. I follow each commit, every action and comment at OpenGothic project. 
I do it so much, that I can't stand to wait for each new version release! 

I have to to admit I am little bit concerned... Since approximately 3 weeks commits started to appear less frequently ( 7-8-9-10 days compared to every day ) than they usually do.  I know, that everyone has his/hers own life and nowadays the author may just have less time for OpenGothic development ( work, school, kids etc. ) On the other hand world struggles with serious situation of COVID19 and he could have been hospitalized or worse.... Can anyone just confirm, than everything is fine with the author and the development is continued?

**Comment by Try** [comment]
Hi @czarnyafgan ! Thanks for asking - I'm alright and healthy :) 

> Since approximately 3 weeks commits started to appear less frequently

That kinda normal, since most of the thins are done here, and remaining ticket are explicitly difficult to get done.
Doing camera work atm. That's is quite a difficult thing to get done, specially to deduce how it behaves in vanilla game.
Apart from that I don't want to release a new build before switching project to *.zip save-game format.

Another thing I'm working in background is tesselation: my engine uses spirv-cross to cross-compile Vulkan shaders into Metal/DirectX, Vulkan -> DX12 path is not implemented in compiler.

**Reply by czarnyafgan** [reply]
Oh God... any ETA for next version release ??? I can't stand, literally I can't !!!

**Reply by Try** [reply]
Ideally I want to make next release in November - it's goina be a year since my first post on reddit - good time to do release + write some progress summary.
Also there are few more things I wish to review:
 * save-load issues (effects, state items) 
 * animation smoothing
 * weather effects

**Comment by czarnyafgan** [comment]
Okay, fine. I am waiting patiently. I'll be glad to test new release :)

wt., 26 paź 2021 o 21:32 Try ***@***.***> napisał(a):

> Ideally I want to make next release in November - it's goina be a year
> since my first post on reddit - good time to do release + write some
> progress summary.
> Also there are few more things I wish to review:
>
>    - save-load issues (effects, state items)
>    - animation smoothing
>    - weather effects
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Try/OpenGothic/discussions/197#discussioncomment-1540855>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AT6RVJ6DTO2STUJBSFS2IJTUI4F4RANCNFSM5FR4PDSA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Pozdrawiam,
Jakub Kruszewski
Konstruktor, NVision System Polska
mob. 00 48 602 398 702
***@***.***
